requirements.in

```diff
@@ -1,6 +1,6 @@
 beautifulsoup4
 boto3
-Django==2.2.14
+Django==2.2.15
```

Comment:
Why not jump to Django 3, do we have dependencies that aren't updated yet?

Reply:
one uplift at a time.